.github/workflows/semver.yml

```diff
@@ -0,0 +1,39 @@
+name: "SemVer"
```

Comment:
Is this different from the one in the `github-actions` repository?

Reply:
@ypk46 
Our github-actions repo has private scope and apparently can't be used from a public repo. So our options are adding a new one or making the github-actions public

Reply:
Oh, got it. That's fine by me. If we identify the need to make a `github-actions` public repository later on, then we may revisit this.